book/src/frequently_asked_questions.md

```diff
@@ -39,6 +28,55 @@ Simply put, we are trying to use settings like `secure_cookies` to add constrain
 that you _must_ perform and adhere to best practices - such as having TLS present on your
 communication channels.
 
+This is also why we do not allow the server to start without a TLS certificate being configured.
+
+### Why disallow HTTP (without TLS) between my load balancer and Kanidm?
+
+Because Kanidm is one of the keys to a secure network, and insecure connections to them are not best
+practice. This can allow account hijacking, privilege escalation, credential disclosures, personal
+information leaks and more. The entire path between a client and the server must be protected at all
+times.
+
+## OAuth2
+
+[RFC6819 - OAuth2 Threat Model and Security Considerations](https://www.rfc-editor.org/rfc/rfc6819)
+is a comprehensive and valuable resource discussing the security of OAuth2 and influences OpenID
+Connect as well. In general Kanidm follows and implements many of the recommendations in this
+document, as well as choosing not to implement certain known insecure OAuth2 features.
+
+### Why is disabling PKCE considered insecure?
```

Comment:
Should this section link to https://kanidm.github.io/kanidm/master/integrations/oauth2.html#extended-options-for-legacy-clients ?

Reply:
done